### ISSUE TITLE
New Page Feature in Visual Studio need update

### ISSUE THREAD

**Issue by jingliancui** [dotnet/maui#6213]
### Description

Used VS2022 Preview to Add a page will lead vs build failed.

### Steps to Reproduce

1.Create a MAUI App from vs2022 preview
2.Right click the project → Add new item → select the ContentPage
3.Xamarin.Form namespace using still in the cs file, need remove these namespace.
4.xaml still used xmlns="http://xamarin.com/schemas/2014/forms", need change it to xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
5.the property of the xaml is Embedded Resource, need change it to maui xaml

### Version with bug

Release Candidate 1 (current)

### Last version that worked well

Release Candidate 1 (current)

### Affected platforms

Android, I was *not* able test on other platforms

### Affected platform versions

maybe all

### Did you find any workaround?

3.Xamarin.Form namespace using still in the cs file, need remove these namespace.
4.xaml still used xmlns="http://xamarin.com/schemas/2014/forms", need change it to xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
5.the property of the xaml is Embedded Resource, need change it to maui xaml

**Comment by XamlTest** [comment]
Not repro on Visual Studio Enterprise 17.3.0 Preview 1.0 [32418.321.main]. Sample Project: [6213.zip](https://github.com/dotnet/maui/files/8509733/6213.zip)

**Comment by jsuarezruiz** [comment]
What templare are you using?
![templates-issue](https://user-images.githubusercontent.com/6755973/164001855-57aa0933-0771-49e6-b19b-ee42f02fb007.gif)
Looks like you are using the Xamarin.Forms ContentPage template and not the .NET MAUI ContentPage one.

**Comment by jingliancui** [comment]
@jsuarezruiz Oh, you are right , I didn't take attention to the left side.
The first UI to me from vs creation item dialog is the "C# items". 
So I select "Content Page" directly that the Content Page will create the Xamarin.Forms' version.
Could you make some UX improvement to this issue?
![image](https://user-images.githubusercontent.com/16309962/164031015-fa23c31d-2759-4b51-bdc3-7f43691c5962.png)

**Comment by LanceMcCarthy** [comment]
@Redth I wanted to adding an additional piece of information to this...

When choosing the MAUI ContentPage selection, it will always use the root namespace of the project and not the subfolder the item is being added to.

Steps to Reproduce (.NET MAUI RC1 on VS 2022 Preview 17.2.0 Preview 4.0)
1. Create a new MAUI project  named "MyProject"
2. Add a folder named `Views` to the project
3. Add a new ContentPage to the folder named "DetailsPage"

=> Observe the Page was added to the Views folder, but the namespace is `MyProject` when it should be `MyProject.Views`

==> Workaround: Manually rename the namespace to MyProject.Views in both the XAML and code-behind file.

Note: I suspected this might be happening because there is no existing namespace yet, but this issue occurs even after there is another page in the Views folder with the correct namespace.
